### ISSUE TITLE
Git pull (Question) 

### ISSUE THREAD

**Issue by Sylenss** [Jean-Tinland/simple-bar#211]
Hey jean, Im trying to pull the current changes that you've made recently after cloning your git repo about 20 days ago. Im still new to git, and I don't think when I cloned your repo it made it into a git repository on my machine, should I create one and submit a pull request? Would you mind helping me, I don't want to interfere, with your repo and need some help?

- The error im receiving is 
<img width="827" alt="CleanShot 2021-09-03 at 14 51 09@2x" src="https://user-images.githubusercontent.com/45975952/132053306-bbbf5b38-5dd2-4c59-b013-d99b7e859f04.png">

**Comment by Jean-Tinland** [comment]
Hi, this is simply a merge conflict as you have local changes that would be overwriten if you pull the latest changes.

Could you run a `git diff`, please, so I can see what are those changes (if there is no sensible data in these) and then tell you what to do next?

**Comment by Sylenss** [comment]
Hello, Jean when I run git diff it doesn't show anything for me. I have settings setup on my machine that if this was a git repository, which it is it would show on the right hand side, ill pull up a project to show you what it looks like, for some reason its not showing that simple-bar is a git repo on my machine, from the git clone that I made 20 days ago. 

* Some how simple-bar is not being recognized as a git project at all from the git clone. 

<img width="825" alt="CleanShot 2021-09-03 at 17 52 31@2x" src="https://user-images.githubusercontent.com/45975952/132068607-904ac321-cb94-462f-ad39-f5d16db86e3d.png">

- Here I just pulled my VS-Code theme from my GitHub to show you what a git repository project would look like on my terminal machine 

<img width="826" alt="CleanShot 2021-09-03 at 17 57 02@2x" src="https://user-images.githubusercontent.com/45975952/132068889-aaff3a1a-2679-4fdc-8472-8e1c0244c960.png">
